ical_updater.json

```diff
@@ -1,7 +1,7 @@
 {
     "sensor.ical": {
       "updated_at": "2019-09-10",
-      "version": "0.5",
```

Comment:
you also have to update this here:
https://github.com/tybritten/ical-sensor-homeassistant/blob/master/custom_components/ical/sensor.py#L17